checks/dependency_update_tool.go

```diff
@@ -44,13 +44,12 @@ func DependencyUpdateTool(c *checker.CheckRequest) checker.CheckResult {
 		return checker.CreateRuntimeErrorResult(CheckDependencyUpdateTool, e)
 	}
 
-	// Return raw results.
-	if c.RawResults != nil {
-		c.RawResults.DependencyUpdateToolResults = rawData
-	}
+	// Set the raw results.
+	pRawResults := rawResults(c)
```

Comment:
race condition, remember all these checks run in parallel. can we do this in a way that doesn't modify the check request?

Reply:
we're not modifying the check request. The code assigns like it used to, except that it creates a local pRawResult if there's none provided by the caller.

I'm probably missing something

Reply:
discussed offline: I had skimmed and was confused at what the function was doing. There was no race condition, just a misleading name/signature.